### ISSUE TITLE
[a11y] Checkboxes in new onboarding screens should have titles to be narrated by the screen reader

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/mozilla-vpn-client#8012]
**Important details:**

* The checkbox is narrated just as “checkbox” with no title, which is not descriptive for the users. The user must navigate forward to its description to understand what the checkbox is for, and then go back to check/uncheck it if needed.

**Affected versions:**

* 2.17.0 (2.202309122328);

**Tested Platforms:**

* Windows 10;

**Prerequisites:**

* Have a clean install of the Mozilla VPN

* Have an active account with a subscription made for Mozilla VPN.

* Have the “New Onboarding” option from Developer options - Features list, enabled; 

**Steps to reproduce:**

1. Authenticate with the account from prerequisites
1. Using the keyboard, on the first onboarding screen navigate to the “Data collection and use” checkbox and observe how the element is narrated.
1. Using the keyboard, on the last onboarding screen navigate to the “Connect VPN on startup” checkbox and observe how the element is narrated.

**Expected result:**

1. 3. The checkbox with its title and state (checked/unchecked) is narrated. 

**Actual result:**

1. 3. The checkbox is narrated as “checkbox”, title is missing. The checkbox state is correctly narrated.

**Notes:**

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/VPN-5533)
┆Reporter: Raluca Monica Ilban

**Comment by data-sync-user** [comment]
➤ Raluca Monica Ilban commented:

Verified using Mozilla VPN 2.21.0 (2.202402151707), on Windows and the issue is no longer reproducible. The toggle is narrated as checkbox (https://mozilla-hub.atlassian.net/browse/VPN-6207 ( https://mozilla-hub.atlassian.net/browse/VPN-6207|smart-link ) ), but its title and state are correctly narrated. 

Marking this verified as fixed. 

!VPN-5533_fixed.mp4|width=452,height=840!
